examples/uppy-s3-example/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "@uppy-example/uppy-s3-example",
+  "version": "1.0.0",
+  "description": "Uppy for AWS S3 with a custom Node.js backend for signing URLs",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "aws-sdk": "^2.1225.0",
+    "body-parser": "^1.20.0",
+    "dotenv": "^16.0.2",
```

Can we use the same range that's define eslewhere in the project please? It helps with upgrading.
```suggestion
    "dotenv": "^16.0.0",
```